pkg/operation/seed/seed.go

```diff
@@ -660,3 +672,68 @@ func seedWantsAlertmanager(keys []string, secrets map[string]*corev1.Secret) boo
 	}
 	return false
 }
+
+// deleteControlPlaneDependencyWatchdogs deletes the shoot specific dependency-watchdog
+// deployments in the invidual shoot control-planes in favour of the central deployment
+// in the seed-bootstrap.
+// TODO: This code is to be removed in the next release.
+func deleteControlPlaneDependencyWatchdogs(k8sSeedClient k8sclient.Interface, crClient client.Client) error {
+	var continueToken string
+
+	for {
+		list, err := k8sSeedClient.CoreV1().Namespaces().List(metav1.ListOptions{
```

Comment:
```suggestion
    list := &corev1.NamespaceList{}
    if err := crClient.List(ctx, list); err != nil {
```

Why not using the controller-runtime client?

Reply:
None other than laziness 😉 Will make the change.

Reply:
@rfranzke But I will have to revendor sigs.k8s.io/controller-runtime because `Continue` is not supported in the currently vendored version. Is that ok?

Reply:
Why do we need this? `List()` should give you everything already

Reply:
There were couple of default pagination issues from earlier experience. Don't want take any chances.

Anyway, no need for revendoring. Worked around using [`Raw`](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.2.0-beta.5/pkg/client/options.go#L273).

Reply:
Pushed the changes.